addons/ondevice-controls/src/components/color-picker/HoloColorPicker.js

```diff
@@ -194,7 +196,9 @@ export class HoloColorPicker extends React.PureComponent {
         </View>
         {this.props.hideSliders ? null : (
           <View>
+            <Text style={{ paddingStart: 4, color: '#859499', fontSize: 12 }}>Saturation</Text>
             <Slider value={s} onValueChange={this._onSValueChange} />
+            <Text style={{ paddingStart: 4, color: '#859499', fontSize: 12 }}>Lightness</Text>
```

Comment:
I think this should be Brightness

Reply:
The two commonly used names are HSL and HSV. [Wikipedia](https://en.wikipedia.org/wiki/HSL_and_HSV) HSB is a thing but is less commonly referred to that way in most software. It's also referred to as [hsl in CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl).

<img width="284" alt="image" src="https://user-images.githubusercontent.com/112166/222172541-a16b637e-4b5e-49f6-8d57-97267991e851.png">

Reply:
ah ok cool